shared/types/response-v3.ts

```diff
@@ -115,3 +115,48 @@ export type AttachmentFieldResponseV3 = {
   hasBeenScanned: boolean
   answer: string
 }
+
+/**
+ * AttachmentResponses with additional server injected metadata on email and storage v2+ forms.
+ */
+export type ParsedClearAttachmentFieldResponseV3 = AttachmentFieldResponseV3 & {
+  filename: string
+  content: Buffer
+}
+
+/**
+ * AttachmentResponses with hydrated content from server
+ *
+ * Note: JSON.parse(JSON.stringify(fooBuffer)) does not reconstruct the Buffer object
+ */
+export type HydratedClearAttachmentFieldResponseV3 =
+  AttachmentFieldResponseV3 & {
+    filename: string
+    content: {
+      data: string
+      type: 'buffer'
+    }
+  }
+
+export type ParsedClearAttachmentResponseV3 = Omit<
+  AttachmentResponseV3,
+  'answer'
+> & {
+  answer: ParsedClearAttachmentFieldResponseV3
+}
+
+export type HydratedClearAttachmentResponseV3 = Omit<
+  AttachmentResponseV3,
+  'answer'
+> & {
+  answer: HydratedClearAttachmentFieldResponseV3
+}
+
+export type ParsedClearFormFieldResponseV3 =
+  | Exclude<FieldResponseV3, AttachmentResponseV3>
+  | ParsedClearAttachmentResponseV3
+
+export type ParsedClearFormFieldResponsesV3 = Record<
+  FormFieldDto['_id'],
+  ParsedClearFormFieldResponseV3
+>
```

Comment:
I see this was moved here to be used in the shared `isFieldResponseV3Equal` function but... - see my comment in that file